tests/source_data_validation/conftest.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+import pytest
+from referencing import Registry, Resource
+
+from tests.source_data_validation import (
+    SCHEMAS,
+    remote_configurations,
+)
+
+
+@pytest.fixture(name="remote_configurations")
+def remote_configurations_fixture():
+    return remote_configurations()
+
+
+@pytest.fixture
```

Comment:
How about sharing the fixture between tests?

```suggestion
@pytest.fixture(scope="session")
```